fastf1/plotting.py

```diff
@@ -286,6 +289,12 @@ def driver_color(identifier: str) -> str:
             ratio = fuzz.ratio(identifier, existing_key)
             key_ratios.append((ratio, existing_key))
         key_ratios.sort(reverse=True)
+        if key_ratios[0][0] != 100:
+            _logger.warning(
+                ("Correcting invalid user input "
+                 f"{identifier} to {key_ratios[0][1]}."
```

Comment:
For readability of the output, quotes around the two variables are probably preferable.
``f"'{identifier}' to ...``